### ISSUE TITLE
Unable to create an Entity with `npx vendure add`

### ISSUE THREAD

**Issue by dim-anis** [vendure-ecommerce/vendure#2834]
**Describe the bug**
`npx vendure add` fails on attempting to create an Entity with:
```
◒  Creating entity│
■  Error adding feature "add-entity"
│
■  Error: Attempted to get information from a node that was removed or forgotten.
│
│  Node text: @Entity()
│  export class ScaffoldTranslation
│      extends VendureEntity
│      implements Translation<S...
│      at get compilerNode [as compilerNode] (/Users/dimanis/dev/vendure-shop/node_modules/ts-morph/dist/ts-morph.js:3042:19)
│      at ClassDeclaration.getNameNode (/Users/dimanis/dev/vendure-shop/node_modules/ts-morph/dist/ts-morph.js:9551:63)
│      at ClassDeclaration.getName (/Users/dimanis/dev/vendure-shop/node_modules/ts-morph/dist/ts-morph.js:9557:25)
│      at ClassDeclaration.rename (/Users/dimanis/dev/vendure-shop/node_modules/ts-morph/dist/ts-morph.js:9563:34)
│      at createEntity (/Users/dimanis/dev/vendure-shop/node_modules/@vendure/cli/dist/commands/add/entity/add-entity.js:115:90)
│      at addEntity (/Users/dimanis/dev/vendure-shop/node_modules/@vendure/cli/dist/commands/add/entity/add-entity.js:40:47)
│      at async Object.createNewPlugin [as run] (/Users/dimanis/dev/vendure-shop/node_modules/@vendure/cli/dist/commands/add/plugin/create-new-plugin.js:134:32)
│      at async addCommand (/Users/dimanis/dev/vendure-shop/node_modules/@vendure/cli/dist/commands/add/add.js:47:50)
│      at async Command.<anonymous> (/Users/dimanis/dev/vendure-shop/node_modules/@vendure/cli/dist/cli.js:52:5)
│      at async Command.parseAsync (/Users/dimanis/dev/vendure-shop/node_modules/@vendure/cli/node_modules/commander/lib/command.js:936:5)
```
**To Reproduce**
1. Clean install. 
2. Create a plugin using `npx vendure add`. 
3. Create new Entity with `npx vendure add`.

**Expected behavior**
Entity created.

**Environment (please complete the following information):**
 - @vendure/core version: 2.2.3
 - Nodejs version: 20.12.0
 - Database (mysql/postgres etc): locally hosted postgres running in a container

**Additional context**
Plugin creation through cli works, it just errors out when trying to create an Entity.
Originally I thought it might have something to do with pnpm, so I tried scaffolding both with pnpm and npm. Makes no difference.

**Comment by lzzfnc** [comment]
Same here!

I've just installed @vendure/cli and running it through yarn. 

__Environment:__
- @vendure/cli: ^2.2.3
- Nodejs version: v18.17.1
- Database: postgres

I can successfully create new plugins, but can't add a new entity

**Comment by michaelbromley** [comment]
This is a known issue - already fixed in master and will be resolved in the next patch (v2.2.4) 👍
